Running on paths containing `` [space] throws an error

I've run it on a volume `External\ SSD` when renaming to `ExternalSSD` everything works fine. Here's the stack trace:
```
internal/modules/cjs/loader.js:1032
  throw err;
  ^

Error: Cannot find module '/Volumes/External'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

--- a/.eslintrc
+++ b/.eslintrc
@@ -1,3 +1,13 @@
 {
-  "extends": ["@dkundel/js", "@dkundel/ts", "@dkundel/js/jest"]
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint", "prettier"],
+  "rules": {
+    "prettier/prettier": "error",
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-empty-function": "off"
+  },
+  "extends": [
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended"
+  ]
 }

--- a/README.md
+++ b/README.md
@@ -9,10 +9,10 @@
 
 > Command-line tool to read `.env` files and execute scripts/commands after loading those environment variables
 
-* Uses [`dotenv`](https://npm.im/dotenv) under the hood
-* Easy to configure
-* Flexible command to execute
-* Let's you override existing environment variables
+- Uses [`dotenv`](https://npm.im/dotenv) under the hood
+- Easy to configure
+- Flexible command to execute
+- Let's you override existing environment variables
 
 <p align="center"><img alt="node-env-run example screenshot. Code below in Documentation section" src="https://cdn.rawgit.com/dkundel/node-env-run/5bc67d1a/assets/node-env-run-screenshot.png" height="350"/></p>
 
@@ -129,6 +129,46 @@ However, you can also use it with completely unrelated executables such as pytho
 
 ```bash
 nodenv app.py --exec python
+```
+
+## Caveats & Limitations
+
+### Additional Arguments
+
+If you want to pass additional flags/arguments to the script you are executing using `node-env-run`, you can use the empty `--` argument and follow it with any arguments you'd want to pass. For example:
+
+```bash
+nodenv index.js --exec "ts-node" -- --log-level debug
+```
+
+`--log-level debug` will be passed to `index.js`.
+
+If you want to do the same with a REPL like node or python you'll have to specify `REPL` explictly, due to some parsing behavior of yargs. For example:
+
+```bash
+nodenv REPL --exec node -- -e "console.log('hello world!')"
+```
+
+### Using Quotes and Escaping Characters
+
+Using quotes for escaping special characters should generally work out of the box. However, there is one edge case if you are trying to use double quotes (`"`) inside and want to preserve it. In that case you'll have to double escape it due to some inner workings of Node.js. For example:
+
+```bash
+nodenv REPL --exec echo -- 'A common greeting is "Hello World"'
+# outputs: A common greeting is Hello World
+
+nodenv REPL --exec echo -- 'A common greeting is \"Hello World\"'
+# outputs: A common greeting is "Hello World"
+```
+
+Similarly if you want to avoid variables to be interpolated you'll have to escape the `$` separately. For example:
+
+```bash
+nodenv REPL --exec echo -- '$PATH'
+# outputs your actual values stored in $PATH
+
+nodenv REPL --exec echo -- '\$PATH'
+# outputs: $PATH
 ```
 
 ## Contributors

--- a/__mocks__/fs.ts
+++ b/__mocks__/fs.ts
@@ -5,7 +5,7 @@ type FileContentDict = Dictionary<string>;
 type DirectoryListing = Dictionary<string[]>;
 
 const fs: any = jest.genMockFromModule('fs');
-const originalFs = require.requireActual('fs');
+const originalFs = jest.requireActual('fs');
 
 let mockFiles: DirectoryListing = {};
 let allMockFiles: FileContentDict = {};

--- a/bin/node-env-run.ts
+++ b/bin/node-env-run.ts
@@ -13,14 +13,9 @@ const debug = Debug('node-env-run');
  * IMPORTANT: By default it will exit the process when child process exits
  * @param  {string} cmd The command to execute
  * @param  {string[]} cmdArgs An array of arguments to pass to the command
- * @param  {boolean} shouldExit Whether the parent process should finish when child finishes
  * @returns {void}
  */
-function runCommand(
-  cmd: string,
-  cmdArgs: string[],
-  shouldExit: boolean = true
-): void {
+function runCommand(cmd: string, cmdArgs: string[]): void {
   const shell: string | boolean = process.env.SHELL || true;
 
   debug(`Execute command: "${cmd}"`);

--- a/bin/node-env-run.ts
+++ b/bin/node-env-run.ts
@@ -1,11 +1,10 @@
 #!/usr/bin/env node
 
-import { spawn } from 'child_process';
+import { spawn } from 'cross-spawn';
 import * as Debug from 'debug';
 import { start } from 'repl';
-
 import { init, parseArgs } from '../lib/cli';
-import { constructNewArgv } from '../lib/utils';
+import { escapeArguments } from '../lib/utils';
 
 const debug = Debug('node-env-run');
 
@@ -24,34 +23,36 @@ function runCommand(
 ): void {
   const shell: string | boolean = process.env.SHELL || true;
 
-  debug(`Execute command: ${cmd}`);
-  debug(`Using arguments: ${cmdArgs}`);
-  debug(`Using shell: ${shell}`);
+  debug(`Execute command: "${cmd}"`);
+  debug(`Using arguments: "%o"`, cmdArgs);
+  debug(`Using shell: "${shell}"`);
   const child = spawn(cmd, cmdArgs, {
     shell,
     stdio: 'inherit',
     env: process.env,
   });
-  child.on('exit', code => {
+  child.on('exit', (code: number) => {
     debug(`Child process exit with code ${code}`);
     process.exit(code);
   });
 }
 
+debug(`Raw args: %o`, process.argv);
 const args = parseArgs(process.argv);
-debug(`Parsed args: [${args.program._.join(', ')}]`);
+debug(`Parsed args: %o`, args.program._);
 debug(`Parsed script: "${args.script}"`);
 const cli = init(args);
 if (cli.isRepl) {
-  if (cli.node) {
+  if (cli.node && args.program.newArguments.length === 0) {
     start({});
   } else {
-    runCommand(args.program.exec, []);
+    const cmdArgs = escapeArguments(args.program.newArguments);
+    runCommand(args.program.exec, cmdArgs);
   }
 } else if (cli.script !== undefined) {
   const cmd = args.program.exec;
-  const cmdArgs = [cli.script, ...args.program.newArguments];
-  runCommand(args.program.exec, cmdArgs);
+  const cmdArgs = escapeArguments([cli.script, ...args.program.newArguments]);
+  runCommand(cmd, cmdArgs);
 } else {
   console.error(cli.error);
   process.exit(1);

--- a/lib/__tests__/run-invalid-script.ts
+++ b/lib/__tests__/run-invalid-script.ts
@@ -6,9 +6,8 @@ jest.mock('./main.js', () => {}, { virtual: true });
 console.error = jest.fn();
 
 import { resolve } from 'path';
-
-import { getScriptToExecute, setEnvironmentVariables } from '../utils';
 import { init, parseArgs } from '../cli';
+import { getScriptToExecute, setEnvironmentVariables } from '../utils';
 
 const __setScriptToExecute = require('../utils').__setScriptToExecute;
 const __setMockFiles = require('fs').__setMockFiles;
@@ -36,7 +35,7 @@ describe('test command without necessary parameters', () => {
     const cli = init(parseArgs(CMD));
     expect(cli.isRepl).toBeFalsy();
     if (cli.isRepl === false) {
-      expect(cli.error.message).toBe('Failed to determine script to execute');
+      expect(cli?.error?.message).toBe('Failed to determine script to execute');
     }
   });
 

--- a/lib/__tests__/run-missing-env-file.ts
+++ b/lib/__tests__/run-missing-env-file.ts
@@ -5,10 +5,8 @@ jest.mock('./main.js', () => {}, { virtual: true });
 // turn off error logs
 console.error = jest.fn();
 
-import { resolve } from 'path';
-
-import { getScriptToExecute, setEnvironmentVariables } from '../utils';
 import { init, parseArgs } from '../cli';
+import { getScriptToExecute, setEnvironmentVariables } from '../utils';
 
 const __setScriptToExecute = require('../utils').__setScriptToExecute;
 const __setMockFiles = require('fs').__setMockFiles;

--- a/lib/__tests__/utils.ts
+++ b/lib/__tests__/utils.ts
@@ -11,7 +11,11 @@ jest.mock(
 // turn off error logs
 console.error = jest.fn();
 
-import { getScriptToExecute, setEnvironmentVariables } from '../utils';
+import {
+  escapeArguments,
+  getScriptToExecute,
+  setEnvironmentVariables,
+} from '../utils';
 
 const __setMockFiles = require('fs').__setMockFiles;
 
@@ -76,5 +80,32 @@ describe('test setEnvironmentVariables', () => {
     setEnvironmentVariables(toOverride, undefined);
     expect(process.env.TEST_ONE).toBe('hello');
     expect(process.env.TEST_TWO).toBe('foo');
+  });
+});
+
+describe('escapeArguments', () => {
+  test('keeps regular arguments untouched', () => {
+    const result = escapeArguments(['--verbose', '-l', 'debug']);
+    expect(result).toEqual(['--verbose', '-l', 'debug']);
+  });
+
+  test('keeps regular assignments', () => {
+    const result = escapeArguments(['--greeting=hello', '-l=debug']);
+    expect(result).toEqual([`--greeting=hello`, `-l=debug`]);
+  });
+
+  test('wraps arguments containing spaces', () => {
+    const result = escapeArguments(['--greeting', 'Hey how is it going?']);
+    expect(result).toEqual(['--greeting', `"Hey how is it going?"`]);
+  });
+
+  test('handles single quotes', () => {
+    const result = escapeArguments(['--greeting', `Hey what's up?`]);
+    expect(result).toEqual(['--greeting', `"Hey what's up?"`]);
+  });
+
+  test('does not touch existing double quotes', () => {
+    const result = escapeArguments(['--greeting', `Hey "Dominik"!`]);
+    expect(result).toEqual(['--greeting', `"Hey "Dominik"!"`]);
   });
 });

--- a/lib/__tests__/utils.ts
+++ b/lib/__tests__/utils.ts
@@ -1,4 +1,3 @@
-/* eslint-disable import/no-absolute-path */
 jest.mock('fs');
 jest.mock(
   '/fake/path/to/package.json',

--- a/lib/cli.ts
+++ b/lib/cli.ts
@@ -3,7 +3,6 @@ import * as Debug from 'debug';
 import * as dotenv from 'dotenv';
 import * as fs from 'fs';
 import * as path from 'path';
-import * as pkginfo from 'pkginfo';
 import * as yargs from 'yargs';
 import {
   EnvironmentDictionary,
@@ -32,7 +31,6 @@ export type Cli =
 
 const debug = Debug('node-env-run');
 const cwd = process.cwd();
-const pkg = pkginfo(module);
 
 const usageDescription = stripIndent`
   Runs the given script with set environment variables. 
@@ -49,7 +47,7 @@ const usageDescription = stripIndent`
  */
 export function parseArgs(argv: string[]): CliArgs {
   const result = yargs
-    .usage('$0 [script]', usageDescription, yargs => {
+    .usage('$0 [script]', usageDescription, (yargs) => {
       yargs.positional('script', {
         describe: 'the file that should be executed',
         type: 'string',

--- a/lib/cli.ts
+++ b/lib/cli.ts
@@ -1,14 +1,13 @@
+import { stripIndent } from 'common-tags';
+import * as Debug from 'debug';
+import * as dotenv from 'dotenv';
 import * as fs from 'fs';
 import * as path from 'path';
-import * as yargs from 'yargs';
-import * as dotenv from 'dotenv';
-import * as Debug from 'debug';
 import * as pkginfo from 'pkginfo';
-import { stripIndent, stripIndentTransformer } from 'common-tags';
-
+import * as yargs from 'yargs';
 import {
-  getScriptToExecute,
   EnvironmentDictionary,
+  getScriptToExecute,
   setEnvironmentVariables,
 } from './utils';
 
@@ -106,6 +105,8 @@ export function parseArgs(argv: string[]): CliArgs {
   const script: string | undefined = result.script;
   result.newArguments = result._;
 
+  debug('Yargs Result %o', result);
+
   return { program: result, script };
 }
 
@@ -133,7 +134,7 @@ export function init(args: CliArgs): Cli {
 
   setEnvironmentVariables(envValues, program.force);
 
-  if (!script) {
+  if (!script || script === 'REPL') {
     const node = args.program.exec === undefined;
     return { isRepl: true, node };
   }

--- a/lib/utils.ts
+++ b/lib/utils.ts
@@ -2,7 +2,9 @@ import * as Debug from 'debug';
 import * as fs from 'fs';
 import { resolve } from 'path';
 
-export type EnvironmentDictionary = { [key: string]: any };
+export type EnvironmentDictionary = {
+  [key: string]: string | undefined;
+};
 
 const debug = Debug('node-env-run');
 
@@ -46,15 +48,16 @@ export function getScriptToExecute(script: string, cwd: string): string | null {
  */
 export function setEnvironmentVariables(
   readValues: EnvironmentDictionary,
-  force: boolean = false
+  force = false
 ): void {
   if (force) {
     debug('Force overriding enabled');
   }
 
-  const envKeysToSet = Object.keys(readValues).filter(key => {
+  const envKeysToSet = Object.keys(readValues).filter((key) => {
     if (force && typeof readValues[key] !== 'undefined') {
-      if (typeof readValues[key] === 'string' && readValues[key].length === 0) {
+      const val = readValues[key];
+      if (typeof val === 'string' && val.length === 0) {
         debug(`Not overriding ${key}`);
         return false;
       }
@@ -66,11 +69,13 @@ export function setEnvironmentVariables(
     return !process.env[key];
   });
 
-  envKeysToSet.forEach(key => {
+  envKeysToSet.forEach((key) => {
     process.env[key] = readValues[key];
   });
 
-  debug(`Set the env variables: ${envKeysToSet.map(k => `"${k}"`).join(',')}`);
+  debug(
+    `Set the env variables: ${envKeysToSet.map((k) => `"${k}"`).join(',')}`
+  );
 }
 
 /**
@@ -100,8 +105,8 @@ const REGEX_NOT_REGULAR_CHARACTER = new RegExp(
  *
  * @param args list of arguments to pass to spawn
  */
-export function escapeArguments(args: string[]) {
-  const escapedArguments = args.map(arg => {
+export function escapeArguments(args: string[]): string[] {
+  const escapedArguments = args.map((arg) => {
     if (arg.match(REGEX_NOT_REGULAR_CHARACTER)) {
       return `"${arg}"`;
     }

--- a/lib/utils.ts
+++ b/lib/utils.ts
@@ -1,6 +1,6 @@
+import * as Debug from 'debug';
 import * as fs from 'fs';
 import { resolve } from 'path';
-import * as Debug from 'debug';
 
 export type EnvironmentDictionary = { [key: string]: any };
 
@@ -87,4 +87,25 @@ export function constructNewArgv(
 ): string[] {
   const [node] = currentArgv;
   return [node, script, ...newArguments.split(' ')];
+}
+
+const REGULAR_SHELL_CHARACTERS = ['a-z', 'A-Z', '1-9', '-', '_', '/', ':', '='];
+const REGEX_NOT_REGULAR_CHARACTER = new RegExp(
+  `[^${REGULAR_SHELL_CHARACTERS.join('')}]`
+);
+
+/**
+ * Wraps arguments that contain characters that need to be escaped into quotes
+ * That way they will be passed correctly to the spawn command
+ *
+ * @param args list of arguments to pass to spawn
+ */
+export function escapeArguments(args: string[]) {
+  const escapedArguments = args.map(arg => {
+    if (arg.match(REGEX_NOT_REGULAR_CHARACTER)) {
+      return `"${arg}"`;
+    }
+    return arg;
+  });
+  return escapedArguments;
 }

--- a/package.json
+++ b/package.json
@@ -21,9 +21,10 @@
     "@types/common-tags": "^1.4.0",
     "@types/debug": "^0.0.29",
     "@types/dotenv": "^4.0.0",
-    "@types/node": "^7.0.18",
+    "@types/node": "^10.17.28",
     "@types/yargs": "^11.0.0",
     "common-tags": "^1.7.2",
+    "cross-spawn": "^7.0.3",
     "debug": "^2.6.9",
     "dotenv": "^4.0.0",
     "pkginfo": "^0.4.1",
@@ -47,6 +48,7 @@
   "devDependencies": {
     "@dkundel/eslint-config-js": "^2.0.0",
     "@dkundel/eslint-config-ts": "^2.0.0",
+    "@types/cross-spawn": "^6.0.2",
     "@types/jest": "^19.2.3",
     "all-contributors-cli": "^4.11.1",
     "eslint": "^4.19.1",
@@ -54,7 +56,7 @@
     "npm-run-all": "^4.0.2",
     "prettier": "^1.11.1",
     "ts-jest": "^20.0.3",
-    "typescript": "^2.7.2"
+    "typescript": "^3.9.7"
   },
   "jest": {
     "transform": {

--- a/package.json
+++ b/package.json
@@ -19,16 +19,16 @@
   "license": "MIT",
   "dependencies": {
     "@types/common-tags": "^1.4.0",
-    "@types/debug": "^0.0.29",
-    "@types/dotenv": "^4.0.0",
+    "@types/debug": "^4.1.5",
+    "@types/dotenv": "^8.2.0",
     "@types/node": "^10.17.28",
-    "@types/yargs": "^11.0.0",
+    "@types/yargs": "^15.0.5",
     "common-tags": "^1.7.2",
     "cross-spawn": "^7.0.3",
-    "debug": "^2.6.9",
-    "dotenv": "^4.0.0",
+    "debug": "^4.1.1",
+    "dotenv": "^8.2.0",
     "pkginfo": "^0.4.1",
-    "yargs": "^13.1.0"
+    "yargs": "^15.4.1"
   },
   "bin": {
     "nodenv": "./dist/bin/node-env-run.js",
@@ -49,13 +49,13 @@
     "@dkundel/eslint-config-js": "^2.0.0",
     "@dkundel/eslint-config-ts": "^2.0.0",
     "@types/cross-spawn": "^6.0.2",
-    "@types/jest": "^19.2.3",
-    "all-contributors-cli": "^4.11.1",
-    "eslint": "^4.19.1",
-    "jest": "^20.0.4",
+    "@types/jest": "^26.0.7",
+    "all-contributors-cli": "^6.17.0",
+    "eslint": "^7.5.0",
+    "jest": "^26.1.0",
     "npm-run-all": "^4.0.2",
-    "prettier": "^1.11.1",
-    "ts-jest": "^20.0.3",
+    "prettier": "^2.0.5",
+    "ts-jest": "^26.1.4",
     "typescript": "^3.9.7"
   },
   "jest": {

--- a/package.json
+++ b/package.json
@@ -46,12 +46,14 @@
     "test": "jest"
   },
   "devDependencies": {
-    "@dkundel/eslint-config-js": "^2.0.0",
-    "@dkundel/eslint-config-ts": "^2.0.0",
     "@types/cross-spawn": "^6.0.2",
     "@types/jest": "^26.0.7",
+    "@typescript-eslint/eslint-plugin": "^3.7.1",
+    "@typescript-eslint/parser": "^3.7.1",
     "all-contributors-cli": "^6.17.0",
     "eslint": "^7.5.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.4",
     "jest": "^26.1.0",
     "npm-run-all": "^4.0.2",
     "prettier": "^2.0.5",
@@ -59,16 +61,7 @@
     "typescript": "^3.9.7"
   },
   "jest": {
-    "transform": {
-      "^.+\\.tsx?$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
-    },
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx)$",
-    "moduleFileExtensions": [
-      "ts",
-      "tsx",
-      "json",
-      "js"
-    ],
+    "preset": "ts-jest",
     "testPathIgnorePatterns": [
       "<rootDir>/dist/",
       "<rootDir>/node_modules/"

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,37 +1,39 @@
 {
   "compilerOptions": {
-    /* Basic Options */                       
-    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'commonjs', 'amd', 'system', 'umd' or 'es2015'. */
+    /* Basic Options */
+
+    "target": "es2015" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'commonjs', 'amd', 'system', 'umd' or 'es2015'. */,
     // "lib": [],                             /* Specify library files to be included in the compilation:  */
     // "allowJs": true,                       /* Allow javascript files to be compiled. */
     // "checkJs": true,                       /* Report errors in .js files. */
     // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
-    "declaration": true,                   /* Generates corresponding '.d.ts' file. */
+    "declaration": true /* Generates corresponding '.d.ts' file. */,
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "./dist",                        /* Redirect output structure to the directory. */
+    "outDir": "./dist" /* Redirect output structure to the directory. */,
     // "rootDir": "./",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
-    "removeComments": true,                /* Do not emit comments to output. */
+    "removeComments": true /* Do not emit comments to output. */,
     // "noEmit": true,                        /* Do not emit outputs. */
     // "importHelpers": true,                 /* Import emit helpers from 'tslib'. */
     // "downlevelIteration": true,            /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
     // "isolatedModules": true,               /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
-                                              
-    /* Strict Type-Checking Options */        
-    "strict": true                            /* Enable all strict type-checking options. */
+
+    /* Strict Type-Checking Options */
+
+    "strict": true /* Enable all strict type-checking options. */
     // "noImplicitAny": true,                 /* Raise error on expressions and declarations with an implied 'any' type. */
     // "strictNullChecks": true,              /* Enable strict null checks. */
     // "noImplicitThis": true,                /* Raise error on 'this' expressions with an implied 'any' type. */
     // "alwaysStrict": true,                  /* Parse in strict mode and emit "use strict" for each source file. */
-                                              
-    /* Additional Checks */                   
+
+    /* Additional Checks */
     // "noUnusedLocals": true,                /* Report errors on unused locals. */
     // "noUnusedParameters": true,            /* Report errors on unused parameters. */
     // "noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
     // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
-                                              
-    /* Module Resolution Options */           
+
+    /* Module Resolution Options */
     // "moduleResolution": "node",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
     // "baseUrl": "./",                       /* Base directory to resolve non-absolute module names. */
     // "paths": {},                           /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
@@ -39,21 +41,18 @@
     // "typeRoots": [],                       /* List of folders to include type definitions from. */
     // "types": [],                           /* Type declaration files to be included in compilation. */
     // "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
-                                              
-    /* Source Map Options */                  
+
+    /* Source Map Options */
     // "sourceRoot": "./",                    /* Specify the location where debugger should locate TypeScript files instead of source locations. */
     // "mapRoot": "./",                       /* Specify the location where debugger should locate map files instead of generated locations. */
     // "inlineSourceMap": true,               /* Emit a single file with source maps instead of having a separate file. */
     // "inlineSources": true,                 /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
-                                              
-    /* Experimental Options */                
+
+    /* Experimental Options */
     // "experimentalDecorators": true,        /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
   },
-  "include": [
-    "lib/**/*",
-    "bin/**/*"
-  ],
+  "include": ["lib/**/*", "bin/**/*"],
   "exclude": [
     "node_modules",
     "dist",

